docs/README.md

```diff
@@ -59,8 +59,16 @@ The [MkDocs reference](https://www.mkdocs.org/user-guide/writing-your-docs/) inc
 
 Review the [Material for MkDocs reference](https://squidfunk.github.io/mkdocs-material/reference/formatting/) for information on the theme and formatting.
 
+### Generating the reference
+
+Our documentation contains an API reference section that is autogenerated from the custom resource definitions of each ACK service. These reference documents need to be re-generated each time there is a change to any of the fields or versions of any of these CRDs.
+
+The `scripts/gen_reference.py` file will attempt to navigate every service controller repository in the `$GOPATH` and build a full model of every custom resource and version based off the CRD base files. It will then output these models into YAML headers placed into markdown files under `contents/reference/<service>/<version>/<resource>.md`, which are then rendered by `mkdocs-macros` snippets in `contents/snippets/`.
```

Comment:
if we are going to integrate this into our postsubmit Prow jobs, is the ProwJob's working tree going to contain all the controller repositories?

Reply:
That's the plan. Attach every service controller to every post submit prow job. It's easy enough to do this programmatically.

Reply:
ack